### ISSUE TITLE
Display custom tags for rooms (and organise them according to those tags)

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-meta#11]
Synapse supports rooms' being assigned custom tags. At present, only Riot Web displays and uses this tag data to organise rooms in the room list on the left.

The scope of this task is to implement this same functionality in Riot iOS and Android.

**Comment by lampholder** [comment]
I don't think this actually is blocked on design - I think the existing redesign documents give enough guidance for implementing this in Android/iOS. @giomfo - can you confirm?

**Comment by turt2live** [comment]
Android: https://github.com/vector-im/riot-android/issues/988

**Comment by progserega** [comment]
I think left slide-able panel must be more functional. Today it is consist "Settings", "Report bug", "version" and other not major things. I think this buttons may be moved to "..."-menu at right of top panel of app.
And add to left slide-able selection of all existing tags of user's rooms.
